backends/arm/test/ops/test_add.py

```diff
@@ -130,7 +131,22 @@ def test_add_tosa_BI(self, test_data: torch.Tensor):
     @parameterized.expand(Add.test_parameters)
     def test_add_u55_BI(self, test_data: torch.Tensor):
         test_data = (test_data,)
-        self._test_add_u55_BI_pipeline(self.Add(), test_data)
+        tester = self._test_add_ethos_BI_pipeline(
+            self.Add(),
+            common.get_u55_compile_spec(permute_memory_to_nhwc=True),
+            test_data,
+        )
+        if common.is_option_enabled("corstone300"):
```

Comment:
how would this run u85 bitstream?

Reply:
The compilespec is for Ethos-U55 and the run_method_and_compare_outpus is only run here for.

Reply:
Right, but this function can be called for both u55 and u85 IIUC